addons/website/views/snippets.xml

```diff
@@ -1442,7 +1442,7 @@
 
     <!-- Anchor Name -->
     <div data-js="anchorName"
-        data-selector=":not(p).oe_structure > *, :not(p)[data-oe-type=html] > *"
+        data-selector=":not(p).oe_structure > section, :not(p).oe_structure > div, :not(p)[data-oe-type=html] > section, :not(p)[data-oe-type=html] > div"
         data-exclude=".modal *">
```

Comment:
All you have to do for part 1 is changing the data-exclude here in website_blog to exclude blog content elements. What you did is even not a stable change, why changing the `*` selector breaking all snippets which do not use section or div? :wink:

Reply:
Indeed, I overrode the data-exclude in the website_blog to add the affected selectors.